src/pudl/transform/ferc1.py

```diff
@@ -2846,21 +2850,17 @@ def process_dbf(self, raw_dbf):
 
         Right now we are just dropping these bad row numbers. Should we actually be
         bumping the whole respondent's row numbers - assuming they reported incorrectly
-        for the whole table?
+        for the whole table? See Issue #471.
```

Comment:
If we use full URLs for issues they'll be easier to access and less ambiguous. GitHub will only auto-render these links in some contexts, and if someone sees them in code they won't get rendered.

```
See: https://github.com/catalyst-cooperative/pudl/issues/471
```